alcs-frontend/src/app/features/application/decision/conditions/condition/condition.component.ts

```diff
@@ -31,16 +44,75 @@ export class ConditionComponent implements OnInit, AfterViewInit {
   isReadMoreClicked = false;
   isReadMoreVisible = false;
   conditionStatus: string = '';
+  isRequireSurveyPlan = false;
+  subdComponent?: DecisionComponentDto;
+  planNumbers: ApplicationDecisionConditionToComponentPlanNumberDto[] = [];
 
-  constructor(private conditionService: ApplicationDecisionConditionService) {}
+  constructor(
+    private conditionService: ApplicationDecisionConditionService,
+    private conditionLotService: ApplicationDecisionComponentToConditionLotService
+  ) {}
 
   ngOnInit() {
     this.updateStatus();
     if (this.condition) {
       this.condition = {
         ...this.condition,
-        componentLabels: this.condition.conditionComponentsLabels?.join(';\n'),
+        componentLabelsStr: this.condition.conditionComponentsLabels?.flatMap((e) => e.label).join(';\n'),
       };
+
+      this.isRequireSurveyPlan = this.condition.type?.code === 'RSPL';
+      this.loadLots();
+      this.loadPlanNumber();
+    }
+  }
+
+  async loadLots() {
+    if (this.condition.components) {
+      const subdComponent = this.condition.components.find((e) => e.applicationDecisionComponentTypeCode === 'SUBD');
+      if (subdComponent && subdComponent.uuid) {
+        const planNumbers = await this.conditionLotService.fetchConditionLots(this.condition.uuid, subdComponent.uuid);
+        subdComponent.lots = subdComponent.lots
+          ?.map(
+            (l) =>
+              ({
+                ...l,
+                planNumbers: planNumbers.find((p) => p.componentLotUuid === l.uuid)?.planNumbers,
+              } as ProposedDecisionLotDto)
+          )
+          .sort((l1, l2) => l1.number - l2.number);
+        this.subdComponent = subdComponent;
+      }
+    }
+  }
+
+  async loadPlanNumber() {
+    const subdComponent = this.condition.components?.find(
+      (e) => e.applicationDecisionComponentTypeCode === APPLICATION_DECISION_COMPONENT_TYPE.SUBD
```

Comment:
e? component doesn't even start with the letter e. Rename to component.

Reply:
done. I think I've got all e, i and l that were added